pkgs/development/libraries/gtk/2.x.nix

```diff
@@ -61,6 +61,10 @@ stdenv.mkDerivation rec {
     ++ optionals cupsSupport [ cups ]
     ++ optionals stdenv.isDarwin [ AppKit Cocoa ];
 
+  preConfigure = lib.optionalString (lib.versionAtLeast stdenv.hostPlatform.darwinMinVersion "11" && stdenv.isDarwin) ''
+    MACOSX_DEPLOYMENT_TARGET=10.16
+  '';
```

Comment:
```suggestion
  preConfigure = if (lib.versionAtLeast stdenv.hostPlatform.darwinMinVersion "11" && stdenv.isDarwin) then ''
    MACOSX_DEPLOYMENT_TARGET=10.16
  '' else null;
```

Reply:
proper way to avoid mass rebuilds. Probably should do a cleaner version in `staging`

Reply:
Updated. Now darwin shows zero rebuilds due to this only really affecting `aarch64-darwin`, not `x86_64-darwin`, which is evaluated by ofborg. Same thing with #123728 and 123729.

Reply:
Isn't that the desired affect? Or am I wrong?

Reply:
It is.